### PR TITLE
Fixed for new placeholder format and Unicode issue with CSV writer

### PR DESCRIPTION
**Pull request by moduloprime**
@meacer PTAL -- Needed to fix processing tool to conform to new placeholder format introduced in #156.

### CODE REVIEW

File: tools/processresults.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def ProcessResults(json_in_file, csv_prefix):
       # Print UnexpectedFormatException and continue, since they are usually
       # due to lack of data for a condition.
       print 'Exception in %s: %s' % (c, e.value)
-  
+
```
Comment (meacer) [inline tools/processresults.py]: Extra space on this line

File: tools/processresults.py [hunk 2]
```diff
@@ -300,9 +303,12 @@ def _ReplaceUrlWithPlaceholder(results):
   """
   for r in results:
     q = r['responses'][0]['question'] # Do replacement in first question only
-    m = re.search('\"Proceed to.*?\"', q)
-    if m:
-      q = q.replace(m.group(0), '\"Proceed to [URL]\"')
-      r['responses'][0]['question'] = q
+    chosenMatch = re.search('\"Proceed to.*?\"', q)
```
Comment (meacer) [inline tools/processresults.py]: Is it possible for the response to include arbitrary characters? For example, could it be something like "Proceed to <script>alert(0)</script>" or something like that?

The client answers would be untrusted but I guess the server would reject malformed answers before reaching here, right?

**Comment by meacer** [comment]
Sorry for the late response, I missed this earlier today. LGTM.

File: tools/processresults.py (hunk 2 above)
Comment (moduloprime) [inline tools/processresults.py]: No, we don't have any input validation happening on the server. Malformed answers are definitely possible. Malformed questions are also possible -- we expect the questions to be the ones our extension generated, but I suppose a malicious user could send malformed questions, too.

I'm not an expert in handling user-generated input, but I figured we're safe as long as we're not executing user-generated strings as code; LMK if you see any of that, or if there are other cases to be concerned about.

**Pull request merged**